src/main/java/org/folio/circulation/domain/notice/session/PatronSessionRecord.java

```diff
@@ -0,0 +1,35 @@
+package org.folio.circulation.domain.notice.session;
+
+import static java.util.Objects.requireNonNull;
+
+public class PatronSessionRecord {
+
+  private final String id;
```

Comment:
Why is this a string rather than a UUID?

Reply:
I can see 4 places where it can be used:
1. Generate UUID when creating record. 
2. Map to json.
3. Map from json.
4. Get id to use it as a **path parameter** (for example, to delete by id).

In cases `2-4` we need this value as `String`. So to avoid `UUID -> String` mapping I decided to have this field as `String`.
What do think?

Reply:
I've been slowly trying to encourage stronger typing in mod-circulation, as it increases the explicitness of expectations around the data and means some things are checked automatically. 

Admittedly there is a long way to go with that. If it is impractical to use a UUID at this point, then it can be deferred. Does the storage API constrain it to a UUID? If so, what will happen if the code uses a non UUID for this value?

For the JSON mapping, either there should be, or methods could be added to the utility classes (JsonPropertyWriter and JsonPropertyReader I think) for doing this.